### DISCUSSION TITLE
Links to logs open Mobile app to file changes

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#53694] [Mobile] [answered]
GitHub Mobile beta

## Steps
1. Open Google Chrome for Android to https://github.com/check-spelling/rust/actions/runs/4793439932/jobs/8525884498
2. Tap ⚙️
3. Tap View raw logs

## Expected results

Raw logs for workflow run

## Actual results

GitHub Mobile app opens to the changes view for the commit

https://user-images.githubusercontent.com/2119212/234309129-fa03d429-aab5-4ee6-b134-dd1494c7f294.mp4

**Comment by taki-on** [comment, marked as answer]
Thank you very much for your report! We will track this issue and have 👀 on it soon.
